includes/admin/tools/class-llms-admin-tool-install-forms.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * LLMS_Admin_Tool_Install_Forms class file
+ *
+ * @package LifterLMS/Admin/Tools/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Admin tool to delete pending batches created by a background processor
+ *
+ * @since [version]
+ */
+class LLMS_Admin_Tool_Install_Forms extends LLMS_Abstract_Admin_Tool {
+
+	/**
+	 * Tool ID.
+	 *
+	 * @var string
+	 */
+	protected $id = 'install-forms';
+
+	/**
+	 * Retrieve a description of the tool
+	 *
+	 * This is displayed on the right side of the tool's list before the button.
+	 *
+	 * @since [version]
+	 *
+	 * @return string
+	 */
+	protected function get_description() {
+		return __( 'Restores LifterLMS user information forms and reusable field blocks to their default versions. Caution: any existing form and field customizations will be lost!', 'lifterlms' );
+	}
+
+	/**
+	 * Retrieve the tool's label
+	 *
+	 * The label is the tool's title. It's displayed in the left column on the tool's list.
+	 *
+	 * @since [version]
+	 *
+	 * @return string
+	 */
+	protected function get_label() {
+		return __( 'Reinstall User Forms', 'lifterlms' );
+	}
+
+	/**
+	 * Retrieve the tool's button text
+	 *
+	 * @since [version]
+	 *
+	 * @return string
+	 */
+	protected function get_text() {
+		return __( 'Reinstall Forms', 'lifterlms' );
+	}
+
+	/**
+	 * Delete all core-created reusable blocks
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	private function delete_reusable_blocks() {
+
+		$blocks = new WP_Query(
+			array(
+				'posts_per_page' => -1,
+				'no_found_rows'  => true,
+				'post_type'      => 'wp_block',
+				'meta_key'       => '_llms_field_id',
+				'meta_compare'   => 'EXISTS',
+			)
+		);
+
+		foreach ( $blocks->posts as $post ) {
```

Comment:
so here we delete even custom fields which are reusable blocks, mmm...
Can't we just remove the default ones?
What's the purpose of removing the custom reusable blocks too?

Reply:
Good catch. We'll have to add additional meta data to properly track them but that shouldn't be a problem.

The reason I'm removing them is because if someone *modifies* one of them and they use the tool to restore to the default the *modifications* will persist after the restoration. In order to fully restore to the default versions they *must* be deleted and recreated.

Reply:
Ok, but you're talking about removing the llms core ones, right?
Here:
> What's the purpose of removing the custom reusable blocks too?
I meant the purpose of removing the reusable blocks which are custom fields.

Reply:
actually @eri-trabiccolo this *does not* delete reusable blocks created by the custom fields add-ons. The `_llms_field_id` is used by the core to keep track of the reusable blocks created by the core and these are the only ones being removed here.

Reusable blocks containing custom fields would have an `_is_llms_field` meta value (handled in the blocks library), but these are not being removed by this tool.

Is this what you were thinking?

Reply:
Oh alright, I confused them. Sorry.